### ISSUE TITLE
Banned genres still in playlist

### ISSUE THREAD

**Issue by KCreek1** [riverscuomo/spotkin_flutter#58]
Despite adding genres to banned list, tracks from that genre are still being added to the playlist. 

I double checked my list and updated manually and still were added. 

![image](https://github.com/user-attachments/assets/2c17d9ca-f87e-4572-8e80-453584fa48e1)

![image](https://github.com/user-attachments/assets/93549416-f00f-4b59-8ee4-bf8c0f469ece)

![image](https://github.com/user-attachments/assets/f623e6fa-d63f-4b6b-be35-8c3609d035ac)

I![image](https://github.com/user-attachments/assets/bd75ca51-10fb-45cd-8598-ea143bfe5ce1)

**Comment by urbainn** [comment]
![image](https://github.com/user-attachments/assets/89de923c-258c-4ef9-a15a-56a7c68d13b2)

the banned genres are correctly sent to the backend server, so this is most likely an issue with https://github.com/riverscuomo/spotkin . I'll try to see if I find anything there.

**Comment by riverscuomo** [comment]
I seem to remember there was some very specific logic there. Like where the
genre falls in the list of artist genres. And even whether the genre
matches exactly or whether it is a substring of one of the artists genres.

**Comment by KCreek1** [comment]
I know for the new-albums script it has this to say about banned genres in the read me:  that it would reject an artist if their **first** genre was found on the reject list. 

But I don't know if that is the same code used for spotkin. 

I did have others on my playlist that had banned genres among the list of genres. So I figured they fell into the group where maybe it wasn't their first genre. I just included the examples above because they only showed one genre and that one was on the reject list.
